services/web/pkg/config/defaults/defaultconfig.go

```diff
@@ -49,7 +49,8 @@ func DefaultConfig() *config.Config {
 				},
 				Apps: []string{"files", "search", "preview", "text-editor", "pdf-viewer", "external", "user-management"},
 				Options: map[string]interface{}{
-					"hideSearchBar": false,
+					"hideSearchBar":        false,
```

Comment:
Please delete `hideSearchBar`. `false` is already the default. No need to keep it here.